references/cli/dryrun.go

```diff
@@ -241,3 +248,140 @@ func readApplicationFromFile(filename string) (*corev1beta1.Application, error)
 	err = json.Unmarshal(fileContent, app)
 	return app, err
 }
+
+func readApplicationFromFiles(cmdOption *DryRunCmdOptions, buff *bytes.Buffer) (*corev1beta1.Application, error) {
+	var app *corev1beta1.Application
+	var policies []*v1alpha1.Policy
+	var wf *wfv1alpha1.Workflow
+	policyNameMap := make(map[string]struct{})
+
+	for _, filename := range cmdOption.ApplicationFiles {
+		fileContent, err := utils.ReadRemoteOrLocalPath(filename, true)
+		if err != nil {
+			return nil, err
+		}
+
+		fileType := filepath.Ext(filename)
+		switch fileType {
+		case ".yaml", ".yml":
+			// only support one object in one yaml file
+			fileContent, err = yaml.YAMLToJSON(fileContent)
```

Comment:
So we only support one object in one yaml file?

Reply:
yes, only handle one object per file for now(yaml format, actually).  Plan to  add more support from up coming improvement

Reply:
Great, I strongly suggest supporting multiple files here. Although it is not necessary to be contained in this PR. And the multiple file loading could be moved into kubevela/pkg and be reused across various projects.